app/api/admin.py

```diff
@@ -1,12 +1,41 @@
 from flask import jsonify, Blueprint
 from app.token import admin_required
-from app.dob import (add_user, user_task, task_delete, update)
+from app.dob import (add_user, user_task, task_delete, update, orgnization,
+                     organisation_details)
 from app.schema import (TaskSchema, InfoSchema,
-                        UserSchema, UpdateSchema)
-from app.util import data_now_json_str
+                        UserSchema, UpdateSchema, OrgnizationSchema,
+                        getInfoSchema)
+from app.util import data_now_json_str, role_valid
+
 admin_bp = Blueprint('admin', __name__, url_prefix='/admin')
 
 
+@admin_bp.route('/get_organisation/<token>', endpoint='get_details',
+                methods=['POST'])
+def get_organisation(token):
+    try:
+        orgnization_schema = OrgnizationSchema()
+        
+        data = data_now_json_str(orgnization_schema)
+        message = orgnization(data, token)
+        return jsonify({"success": True, "message": message}), 200
+    except Exception as err:
+        return jsonify({"success": False, "message": str(err)}), 400
+
+
+@admin_bp.route('/create_orgnization', endpoint='create_orgnization',
+                methods=['POST'])
```

Comment:
Some kind of validation should be there that organization and Admin should create using the link we send vai email.